examples/simple/igraph_assortativity_nominal.c

```diff
@@ -0,0 +1,44 @@
+#include <igraph.h>
+#include <stdio.h>
+
+int main(){
+    igraph_t g;
+    igraph_integer_t nodes = 1000, types = 50;
+    igraph_vector_int_t node_type_vec;
+
+    igraph_matrix_t pref_matrix;
+    igraph_matrix_init(&pref_matrix, types, types);
+
+    igraph_rng_seed(igraph_rng_default(), 42);
+    printf("Randomly generated graph with 1000 nodes and 50 vertex types\n\n");
+
+    /* Generate preference matrix giving connection probabilities for different vertex types */
+    for(igraph_integer_t i = 0; i < types; i++){
+        for(igraph_integer_t j = i; j < types; j++){
+            MATRIX(pref_matrix, i, j) = igraph_rng_get_unif(igraph_rng_default(), 0, 1);
+            MATRIX(pref_matrix, j, i) = MATRIX(pref_matrix, i, j);
+        }
+    }
+
+    for (int i = 0; i < 5; i++) {
+        igraph_real_t assortativity;
+    	igraph_vector_int_init(&node_type_vec, nodes);
```

Comment:
This line has a tab character. Please use only spaces.

Reply:
Sorry. I'll be more careful.